src/index.ts

```diff
@@ -31,7 +31,7 @@ export default class Mnoga {
   private fallback: string = DEFAULT_LOCALE;
   private keyMode: boolean = false;
   private locale?: string;
-  private phrases: Phrases = {};
+  private phrases: { [locale: string]: Phrases } = {};
```

Comment:
Going by the API, I think this is the more correct type. We're assigning `Phrases` to a particular locale key so it's not valid for `this.phrases['en']` to be a string.